common.gypi

```diff
@@ -2,15 +2,7 @@
   'target_defaults': {
     'default_configuration': 'Release',
     'cflags_cc' : [
-      '-std=c++14',
-      # The assumption is that projects based on node-cpp-skel will also
-      # depend on mason packages. Currently (this will change in future mason versions)
-      # mason packages default to being built/linked with the CXX11_ABI=0.
-      # So we need to link this way too. This allows source
-      # compiling your module on any ubuntu version, even the latest
-      # where the CXX11_ABI default has flipped to 1
-      # More details at https://github.com/mapbox/mason/issues/319
-      '-D_GLIBCXX_USE_CXX11_ABI=0'
```

Comment:
Do we want this variable enabled for all node versions now or only for node 16?

Reply:
I only needs to be enabled for node16. For older node versions, it works either way.